core/server/utils/index.js

```diff
@@ -112,7 +112,8 @@ utils = {
     tokens: require('./tokens'),
     sequence: require('./sequence'),
     ghostVersion: require('./ghost-version'),
-    mobiledocConverter: require('./mobiledoc-converter')
+    mobiledocConverter: require('./mobiledoc-converter'),
+    markdownConverter: require('./markdown-converter')
```

Comment:
that's fine, doesn't matter so much. If somebody wants to require `require('../utils')` he has access to `utils. markdownConverter`.